DQM/src/DQM/EcalClusterAnalyzer.cxx

```diff
@@ -279,6 +280,20 @@ void EcalClusterAnalyzer::analyze(const framework::Event& event) {
   histograms_.fill(
       "clusterless_hits_percentage",
       100. * (ecal_rec_hits.size() - clustered_hits) / ecal_rec_hits.size());
+
+  if (!inverse_skim_) {
```

Comment:
Why start with the `not` branch when you are doing both branches anyways? I suggest flipping them to make the code easier to reason about.

```cpp
if (skim_for_zero_clusters_) {
  if (n_ecal_clusters > 0) {
    setStorageHint(framework::HINT_SHOULD_DROP);
  } else {
    setStorageHint(framework::HINT_SHOULD_KEEP);
  }
} else {
  if (n_ecal_clusters > 0) {
    setStorageHint(framework::HINT_SHOULD_KEEP);
  } else {
    setStorageHint(framework::HINT_SHOULD_DROP);
  }
}

Reply:
OK sure, I can do this